backend/db/migrations/20201005202813-addAdminLog.js

```diff
@@ -0,0 +1,32 @@
+const tableName = 'admin_logs'
+
+const { AdminLogActions } = require('../../enums')
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    const isPostgres = queryInterface.sequelize.options.dialect === 'postgres'
+
+    return queryInterface.sequelize.transaction(async () => {
+      await queryInterface.createTable(tableName, {
+        id: {
+          type: Sequelize.INTEGER,
+          autoIncrement: true,
+          primaryKey: true
+        },
+        action: Sequelize.ENUM(AdminLogActions),
+        shop_id: Sequelize.INTEGER,
+        seller_id: Sequelize.INTEGER,
+        data: isPostgres ? Sequelize.JSONB : Sequelize.JSON,
+        created_at: Sequelize.DATE,
+        updated_at: Sequelize.DATE
```

Comment:
Shouldn't this be append-only?

Reply:
Yes, it should never get updated. I'll remove the updated_at.

Reply:
done